### ISSUE TITLE
We need online safety and inclusion puzzles.

### ISSUE THREAD

**Issue by chadsansing** [MozillaFoundation/mpa-cryptomancer-challenge#5]
Help us create puzzles and obstacles related to privacy, security, and inclusion issues like cryptography, data-sharing, and going dark.

These should be adaptable for several different adventures.

Describe your work and/or link to it by commenting below.

**Comment by cryptomancer-actual** [comment]
I'd love to help someone design a prison/dungeon that heavily utilizes crypto-gears... basically, SCADA meets IoT. For example, a prison that uses crypto-gears to open/close prison cells, uses hard to reach shards for scrying (like surveillance cameras), etc. And all connected to a golem. Defeating the prison's controls would require players to enumerate shardnets, bridge those shardnets to CodeBreakers to crack weak keyphrases, and abuse the prison's controls accordingly.

Players would have to either escape the thing from inside a cell, or plan a prison break.

**Comment by BakuDreamer** [comment]
What if : it's a prison, but for ghosts !  How do you put a ghost in prison
?!   Encrypt it's true name so it doesn't know what it is anymore and lock
it's soul key up in a array controller based encryption storage network.
That's what has to be hacked to release the ghost the PCs want.  They have
to make sure not to release the ghost of Ecthrois however.  That would be
very bad.  ( Ecthrois however would owe them ' a solid ' ... )

**Comment by cryptomancer-actual** [comment]
@BakuDreamer Sorry, I moved my comment back to the "adventure" section, as it seemed more appropriate.

**Comment by cryptomancer-actual** [comment]
So, I think one of the simplest puzzles/obstacles that could be included in any session/adventure of Cryptomancer is the challenge of having a conversation with someone while actively under surveillance. Here are some simple scenarios:

Alice and Bob need to share a secret, but Chuck is listening. I'm talking in real life, not on the Shardnet. Maybe writing on scrolls or using the Babel spell (which encrypts one's voice).
-Alice and Bob can't exchange a keyphrase to encrypt their traffic, because Chuck will hear it and be able to decrypt their encrypted communication.
-Alice and Bob can use asymmetric crypto to solve for this. Even if Chuck hears one of their True Names, he won't be able to decrypt a payload encrypted with it.
-But what if they don't even want Chuck to know their True Name (because in Cryptomancer, it can be used to track someone)? Now they have to find another way to share true names. There are spells in the game that support this (Share Sight, Messenger), but it might also include passing notes, etc.

In short, a scene of intrigue will include people talking in secret while under direct surveillance.

**Comment by BakuDreamer** [comment]
So they're not using shards at all ?

**Comment by BakuDreamer** [comment]
This very thing gets addressed in an obscure SF novel.  They're in a
repressive state with not just mass surveillance but continuous
surveillance and the state computers can break any code instantly, so to
communicate something secret, people speak in a kind of Cockney rhyming
slang, but, it has to be invented on the spot, or the computers will
understand it.  I think that's in  ' Limes Inferior ' by Janusz Zajdel (
Polish )

**Comment by BakuDreamer** [comment]
' Koalang '   Not from ' Limes Inferior ' from '  Paradyzja ', same author.  
  
  " Koalang, an artificial poetic language invented by the inhabitants of Paradise to evade the electronic eavesdropping system. Because Paradyzja is a space station, and activity is tracked by automatic cameras and analysed, mostly, by computers, its people created an Aesopian language, which is full of metaphors that are impossible for computers to grasp. The meaning of every sentence depended on the context. For example, "I dreamt about blue angels last night" means "I was visited by the police last night."  The software that analyzes sentences is self-learning. Thus, a phrase that is used to describe something metaphorically should not be used again in the same context.  "

**Comment by chadsansing** [comment]
@BakuDreamer - that's a great idea that could also be tied directly to the importance of managing multiple passwords.

**Comment by cryptomancer-actual** [comment]
Talking offline with @BakuDreamer, he developed an excellent means of exchanging keyphrases while under surveillance. Two characters under deep surveillance meet at a tavern and continue their ongoing game of chess. They make three moves. These moves generate the keyphrases they will use to communicate on the public Shardscape that day. For example, the keyphrase might be "pawn moves, bishop takes pawn, rook takes bishop." Tomorrow, they'll play more chess, get a new keyphrase. To onlookers, it's just a routine. It would take a pretty clever analyst to figure out what's going on.

This is all to say that there are a number of innocent seeming games, pieces of media (books, songs, poems), and other habits that could be used to generate keyphrases in plain sight, without an adversary compromising them.

**Comment by chadsansing** [comment]
Righteous - how can we spread these ideas (the game-based keyphrase generator, the currency adventure) to attract contributors and site hosts to develop multiple ideas further? Will be thinking & working on that this week :)

**Comment by cryptomancer-actual** [comment]
@chadsansing 

1) To attract folks, emphasize a free copy of the game. I'll start pushing it a little harder once the free copy thing is squared away.

2) I'm quite confident that even with the small retinue we have so far, we'll be able to crank out crypto puzzles and hacking obstacles. What we actually *need* IMHO, are just some fiction writers and gamer nerds to draft the "adventure" parts of adventures, and periodically stopping to say [insert security/privacy puzzle here]. Then we can swoop in and fill in those blanks.

**Comment by chadsansing** [comment]
Will tweet accordingly this week -
